pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2195,13 +2195,13 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
             category = CATEGORY_STORAGE_ML,
             doc = "Minimum backlog entries for any cursor before start caching reads"
     )
-    private int managedLedgerMinimumBacklogEntriesForCaching = 1000;
+    private int managedLedgerMinimumBacklogEntriesForCaching = 100;
     @FieldContext(
             dynamic = true,
             category = CATEGORY_STORAGE_ML,
             doc = "Maximum backlog entry difference to prevent caching entries that can't be reused"
     )
-    private int managedLedgerMaxBacklogBetweenCursorsForCaching = 1000;
+    private int managedLedgerMaxBacklogBetweenCursorsForCaching = 10000;
```

Comment:
And here

Reply:
replied in other comment https://github.com/apache/pulsar/pull/22295#issuecomment-2006237183